src/runtime/entries/cloudflare.ts

```diff
@@ -29,6 +29,8 @@ async function handleEvent(event: FetchEvent) {
     body = Buffer.from(await event.request.arrayBuffer());
   }
 
+  globalThis.__env__ = globalThis;
```

Comment:
Should this be moved to a separate issue/PR ?

Reply:
Yes, please. Also i'm wondering why this line is not working in unenv:

https://github.com/unjs/unenv/blob/f4cba4ec85becd2e7f4f4d995c46ee34f8ab49dc/src/runtime/node/process/_process.ts#L163

(by doing above, setting something to `process.env` in cf preset causes updating globalThis)

Reply:
Disabled the tests for now. I'm not entirely sure why it's happening, it appears I'm unable to reproduce the bug outside of the tests 🤔

Reply:
It was because of a wrong Miniflare configuration 🤦🏽 Fixed now